src/wrappers/node.js

```diff
@@ -1 +1,2 @@
+global.FormData = require('form-data');
```

Comment:
should probably wrapped in an `if` statement to prevent overriding existing polyfill, or use it only where needed... but that would require a check a the top of each module (3 I think) since it's a node only lib